examples/se2_inverse.py

```diff
@@ -12,12 +12,11 @@
 from theseus import LieGroupTensor
 from theseus.geometry.lie_group import LieGroup
 
-from theseus.geometry.tests.test_se2 import create_random_se2
 
 rng = torch.Generator()
 rng.manual_seed(0)
-x1 = create_random_se2(1, rng)
-x2 = create_random_se2(1, rng)
+x1 = th.SE2.rand(1, generator=rng)
```

Comment:
Let's make aliases for all of these, as in `th.randse2()`, `th.randse3()`, `th.randpoint2()`, etc.

Reply:
Do you mean we add `theseus.rand_se2()` etc. in `theseus.geometry.utils`?

Reply:
You can define them each in the file as. For example., in `so3.py` you can define `randnso3 = SO3.radn` at the bottom, and make them accessible at `theseus` level in the `__init__.py` files.

Reply:
Will do.

Reply:
Added.